Configuration/TypoScript/setup.txt

```diff
@@ -92,7 +92,12 @@ plugin.tx_dpf {
 			universityCollection = {$plugin.tx_dpf.settings.universityCollection}
 
 			timezone = Europe/Berlin
-        }
+
+			# Settings for the fluid pagination view helper
```

Comment:
Is this a general option or can the comment (or the parameter name) say what this is configuring exactly?

Reply:
I renamed the setting and adapted the comment.